community_topics_workflow.md

```diff
@@ -0,0 +1,63 @@
+- [ ] Adds the topic to the [Board](https://github.com/orgs/ansible-community/projects/2/views/1).
+
+### Discussion stage
+
+- [ ] If the topic is ready to be discussed, the Committee person:
+
+  - [ ] Adds the `discussion_topic` label.
+  - [ ] Moves the topic to the `In Discussion` column on the [Board](https://github.com/orgs/ansible-community/projects/2/views/5).
+  - [ ] Open the discussion by adding a comment asking the Community and the Committee to take part in it.
+- [ ] No sync discussion is needed (there are no blockers, complications, confusion, or impasses).
+
+### Voting stage
+
+- [ ] Depending on the topic's complexity, after 1-2 weeks pass since the discussion was open, the Committee person formulates vote options based on the prior discussion and gives participants reasonable amount of time to propose changes to the options (no longer than a week). The person summarizes the options in a comment and also establishes a date when the vote begins if there are no objections about the options / vote date.
+- [ ] In the vote date, the vote starts with the comment of a Committee person which opens the vote and establishes a date when the vote ends ($CURRENT_DATE + no longer than 14 days).
+- [ ] The Committee person labels the topic with the `active-vote` label and moves the topic to the `Active Vote` column on the [Board](https://github.com/orgs/ansible-community/projects/2/views/5).
+- [ ] The Committee person adds `[Vote ends on $YYYY-MM-DD]` to the beginning of the topic's description.
+
+### Voting result stage
+
+- [ ] The next day after the last day of the vote, the Committee person:
+
+  - [ ] Removes the `active-vote` label.
+  - [ ] Add a comment that the vote ended.
+  - [ ] Changes the beginning of the topic's description to `[Vote ended]`.
+  - [ ] Counts the votes separating ones made by Community and made by Committee and creates a summary comment containing the result.
+  - [ ] Asks another Committee person to count the votes and provide the summary.
+- [ ] Up to the [next](https://github.com/ansible/community/blob/main/meetings/README.md#wednesdays) Community meeting, at least two Steering Committee members count the votes and summarize the result in comments.
+- [ ] The vote's result and the final decision are announced in the Community meeting.
+
+### Implementation stage
+
+- [ ] If the topic implies some actions (if it does not, just mark this as complete), the Committee person:
+
+  - [ ] Assigns the topic to a person responsible for performing the actions.
+  - [ ] Add the `being_implemented` label to the topic.
+- [ ] After the topic is implemented, the assignee:
+
+  - [ ] Comments on the topic that the work is done.
+  - [ ] Removes the `being_implemented` label.
+  - [ ] Add the `implemented` label.
+- [ ] A Committee person moves the topic to the `Resolved` column on the [Board](https://github.com/orgs/ansible-community/projects/2/views/5) and closes the topic.
```

Comment:
I wonder whether some logic could automatically move the issues to the correct category based on the labels (or vice versa, apply labels when moved to a category). Having to do both manually will sooner or later result in errors where folks forget about one of the two.

Reply:
Agreed, will see if we can do something with this

Reply:
You can automate the new GitHub Project Board (Beta) https://docs.github.com/en/issues/trying-out-the-new-projects-experience/automating-projects I'm not aware of any examples. Though I'm going to be talking with GitHub soon, so if you can document what we'd like to achieve I can ask how it could be achieved.

Reply:
@gundalow sounds great!
@felixfontein @GregSutcliffe let's think of what else can be nice to see automated:)

Reply:
I think moving an issue into the right category (in the workflow) based on the labels, and removal of other state labels if a new one is added, would be great.